### ISSUE TITLE
Failed to remove aborted plot jobs

### ISSUE THREAD

**Issue by TobiasJu** [tydeno/Ploto#54]
So my System crashed and now starting Ploto again is not possible.

````
PS C: \ Users \ User \ Downloads \ Ploto-1.0.9.3> Remove-AbortedPlotoJobs
PlotoRemoveAbortedJobs @ 5/21/2021 7:24:38 PM: Found aborted Jobs to be deleted: 345de945-fff5-43a3-8e0d-ae185ad731d7 b470c5e7-d726-40ed-ac15-1a5534692aec 345deff545-fff5-43aec 345deff545-fff5-43aec 345de945-fff5-43aec 345deff545-fff5-43aec 345deff545-fff5-43d3 -8e0d-ae185ad731d7 345de945-fff5-43a3-8e0d-ae185ad731d7 763e19f0-d4fb-4AAA-89f0-e6c17a18b1af 6e8732fc-078f-4b2d-a7bb-930a75c8849e bcfa89e1-8d3d-463d-8b3d-fc127b44660b b6cf183c-3393-4eb3-ad0e-7826a95d43ea 93bcc182 -24a7-42a3-8eea-9f195b3bf43a 3c06a3ad-ffcf-44bc-ae6e-1a22f796a4f4 dbbf13b3-efd0-40dc-bf39-5424ae7ebaf4
PlotoRemoveAbortedJobs @ 05/21/2021 7:24:38 PM: Cleaning up ...
PlotoStopJob @ 05/21/2021 7:24:38 PM: ERROR: The "Id" parameter cannot be bound. The value "None" cannot be converted to the type "System.Int32". Error: "The input string is in the wrong format."
Get-ChildItem: The argument cannot be processed because the value of the "path" argument is NULL. Change the value of the "path" argument to a value not equal to
ZERO.
In C: \ Users \ User \ Downloads \ Ploto-1.0.9.3 \ Ploto.psm1: 1191 characters: 25
+ $ FileArrToDel = Get-ChildItem $ job.TempDrive | Where-Object {...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo: InvalidArgument: (:) [Get-ChildItem], PSArgumentNullException
    + FullyQualifiedErrorId: ArgumentNull, Microsoft.PowerShell.Commands.GetChildItemCommand

```

```
PS C:\Users\User\Downloads\Ploto-1.0.9.3> Get-PlotoJobs


JobId              : 4967f0c4-37de-422b-ac8a-bf400eda956d
Status             : 3.1
StartTime          : 05/21/2021 05:30:16
TempDrive          : G:
T2Drive            : 2
OutDrive           : J:
PID                : 8348
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t G:\ -d J:\
PlotId             : 4ffb25c03b0e6bf53358263e6bc68c87813afeeb23fbf3e5a1da28992220ea05
LogPath            : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_5_30_4967f0c4-37de-422b-ac8a-bf400eda956d_Tmp-G_Out-J.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_5_30_4967f0c4-37de-422b-ac8a-bf400eda956d_Tmp-G_Out-J@Stat.txt
CompletionTime     : Still in progress
CompletionTimeP1   : 19933.990 seconds. CPU (100.420%) Fri May 21 11:02:30 2021
CompletionTimeP2   : 5560.560 seconds. CPU (77.430%) Fri May 21 12:35:11 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : 345de945-fff5-43a3-8e0d-ae185ad731d7
Status             : Aborted
StartTime          : 05/21/2021 06:05:40
TempDrive          : G:
T2Drive            : 2
OutDrive           : J:
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t G:\ -d J:\
PlotId             : 7b0dec2453163fa451451d3df14fd33a169f289d49be0b0e0bbeadbf6ddf45e6
LogPath            : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_6_5_345de945-fff5-43a3-8e0d-ae185ad731d7_Tmp-G_Out-J.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_6_5_345de945-fff5-43a3-8e0d-ae185ad731d7_Tmp-G_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 19808.190 seconds. CPU (99.180%) Fri May 21 11:35:49 2021
CompletionTimeP2   : 4992.051 seconds. CPU (80.930%) Fri May 21 12:59:01 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : b470c5e7-d726-40ed-ac15-1a5534692aec
Status             : Aborted
StartTime          : 05/21/2021 04:54:58
TempDrive          : G:
T2Drive            : 2
OutDrive           : J:
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t G:\ -d J:\
PlotId             : 0cb160c2cd465cbb299149d077924620c3bbb114aea949bbd31926e0fb892a4a
LogPath            : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_4_54_b470c5e7-d726-40ed-ac15-1a5534692aec_Tmp-G_Out-J.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_4_54_b470c5e7-d726-40ed-ac15-1a5534692aec_Tmp-G_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 19983.223 seconds. CPU (101.540%) Fri May 21 10:28:02 2021
CompletionTimeP2   : 6255.967 seconds. CPU (72.830%) Fri May 21 12:12:18 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : 345de945-fff5-43a3-8e0d-ae185ad731d7
Status             : Aborted
StartTime          : 05/21/2021 06:05:40
TempDrive          :
T2Drive            :
OutDrive           :
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t G:\ -d J:\
PlotId             : 9be16635ab2ac6628889e392bab0892c8b536df51e46e409e2f33b74c014a535
LogPath            : C:\Users\User\.chia\mainnet\plotter\plotter_log_991cd17e-40c2-48bd-8b46-d8f957322373.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_6_5_345de945-fff5-43a3-8e0d-ae185ad731d7_Tmp-G_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 16392.320 seconds. CPU (117.610%) Fri May 21 08:03:03 2021
CompletionTimeP2   : 7186.492 seconds. CPU (79.330%) Fri May 21 10:02:49 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : 345de945-fff5-43a3-8e0d-ae185ad731d7
Status             : Aborted
StartTime          : 05/21/2021 06:05:40
TempDrive          :
T2Drive            :
OutDrive           :
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t G:\ -d J:\
PlotId             : adaf93ac5764251d1c93885771e5c6aa7abd69c421ed05dfae8c59f54dd24103
LogPath            : C:\Users\User\.chia\mainnet\plotter\plotter_log_8b148f57-a4d4-4fcc-bf46-f3031fad15ad.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_6_5_345de945-fff5-43a3-8e0d-ae185ad731d7_Tmp-G_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 12918.556 seconds. CPU (134.310%) Fri May 21 06:32:19 2021
CompletionTimeP2   : 7361.871 seconds. CPU (87.540%) Fri May 21 08:35:01 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : 345de945-fff5-43a3-8e0d-ae185ad731d7
Status             : Aborted
StartTime          : 05/21/2021 06:05:40
TempDrive          :
T2Drive            :
OutDrive           :
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t G:\ -d J:\
PlotId             : e08c186f1ef04d17e0760a0ffed2abbed1f4cd94c8cc7eac24dd6f0b6c3508d5
LogPath            : C:\Users\User\.chia\mainnet\plotter\plotter_log_1e7d5c9d-c691-4070-b73f-36cc4fd3d657.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_6_5_345de945-fff5-43a3-8e0d-ae185ad731d7_Tmp-G_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 15336.262 seconds. CPU (119.820%) Fri May 21 07:12:14 2021
CompletionTimeP2   : 7950.994 seconds. CPU (79.460%) Fri May 21 09:24:45 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : 763e19f0-d4fb-4aaa-89f0-e6c17a18b1af
Status             : Aborted
StartTime          : 05/21/2021 04:04:40
TempDrive          : F:
T2Drive            : 2
OutDrive           : J:
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t F:\ -d J:\
PlotId             : e9dfa706e0ea838e9c7839bf708c8d6c1c9422cdbcb5f58218f7134e6b6af284
LogPath            : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_4_4_763e19f0-d4fb-4aaa-89f0-e6c17a18b1af_Tmp-F_Out-J.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_4_4_763e19f0-d4fb-4aaa-89f0-e6c17a18b1af_Tmp-F_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 21137.146 seconds. CPU (94.700%) Fri May 21 09:56:58 2021
CompletionTimeP2   : 7056.997 seconds. CPU (68.190%) Fri May 21 11:54:35 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : 6e8732fc-078f-4b2d-a7bb-930a75c8849e
Status             : Aborted
StartTime          : 05/21/2021 03:44:34
TempDrive          : G:
T2Drive            : 2
OutDrive           : J:
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t G:\ -d J:\
PlotId             : cc917778f46bcb9e06dc2e795df1f3632c8606887a8a29c4c688b2fa4117dfd9
LogPath            : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_3_44_6e8732fc-078f-4b2d-a7bb-930a75c8849e_Tmp-G_Out-J.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_3_44_6e8732fc-078f-4b2d-a7bb-930a75c8849e_Tmp-G_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 17860.525 seconds. CPU (110.010%) Fri May 21 08:42:15 2021
CompletionTimeP2   : 7430.422 seconds. CPU (74.620%) Fri May 21 10:46:06 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : bcfa89e1-8d3d-463d-8b3d-fc127b44660b
Status             : Aborted
StartTime          : 05/21/2021 03:29:31
TempDrive          : F:
T2Drive            : 2
OutDrive           : J:
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t F:\ -d J:\
PlotId             : b168b944a893dfa6de7fb3bd9ce436d61b54f674e7a6f4b4e58d75fbd1eb9fbb
LogPath            : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_3_29_bcfa89e1-8d3d-463d-8b3d-fc127b44660b_Tmp-F_Out-J.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_3_29_bcfa89e1-8d3d-463d-8b3d-fc127b44660b_Tmp-F_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 19526.176 seconds. CPU (100.780%) Fri May 21 08:54:58 2021
CompletionTimeP2   : 9060.284 seconds. CPU (65.030%) Fri May 21 11:25:58 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : b6cf183c-3393-4eb3-ad0e-7826a95d43ea
Status             : Aborted
StartTime          : 05/21/2021 02:54:24
TempDrive          : F:
T2Drive            : 2
OutDrive           : J:
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t F:\ -d J:\
PlotId             : dfafab1b4aca8538c999177d419608303d1490db557e0b3c8f4ae92753a470f6
LogPath            : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_2_54_b6cf183c-3393-4eb3-ad0e-7826a95d43ea_Tmp-F_Out-J.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_2_54_b6cf183c-3393-4eb3-ad0e-7826a95d43ea_Tmp-F_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 17008.946 seconds. CPU (108.430%) Fri May 21 07:37:54 2021
CompletionTimeP2   : 9337.351 seconds. CPU (67.690%) Fri May 21 10:13:31 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : 93bcc182-24a7-42a3-8eea-9f195b3bf43a
Status             : Aborted
StartTime          : 05/21/2021 04:39:52
TempDrive          : F:
T2Drive            : 2
OutDrive           : J:
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t F:\ -d J:\
PlotId             : 9e8871b09957f44246b1b6623253eb78300ada9be8618755732fafa0f971b60b
LogPath            : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_4_39_93bcc182-24a7-42a3-8eea-9f195b3bf43a_Tmp-F_Out-J.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_4_39_93bcc182-24a7-42a3-8eea-9f195b3bf43a_Tmp-F_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 22699.884 seconds. CPU (89.440%) Fri May 21 10:58:13 2021
CompletionTimeP2   : 6773.420 seconds. CPU (62.940%) Fri May 21 12:51:06 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : 3c06a3ad-ffcf-44bc-ae6e-1a22f796a4f4
Status             : Aborted
StartTime          : 05/21/2021 04:19:44
TempDrive          : G:
T2Drive            : 2
OutDrive           : J:
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t G:\ -d J:\
PlotId             : eb78eac08d0f28917c00e6c53fc912fee614adea20f7050ff1c2cbaf0648cf29
LogPath            : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_4_19_3c06a3ad-ffcf-44bc-ae6e-1a22f796a4f4_Tmp-G_Out-J.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_4_19_3c06a3ad-ffcf-44bc-ae6e-1a22f796a4f4_Tmp-G_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 19210.637 seconds. CPU (104.070%) Fri May 21 09:39:55 2021
CompletionTimeP2   : 6726.368 seconds. CPU (74.850%) Fri May 21 11:32:02 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :

JobId              : dbbf13b3-efd0-40dc-bf39-5424ae7ebaf4
Status             : Aborted
StartTime          : 05/21/2021 03:09:27
TempDrive          : G:
T2Drive            : 2
OutDrive           : J:
PID                : None
PlotSizeOnTempDisk : 0,00 GB
PlotSizeOnOutDisk  : 0,00 GB
ArgumentList       : plots create -k 32 -b 3390 -r 2 -t G:\ -d J:\
PlotId             : b323c48eac9e635980f8b4b365511e3d7fcd28f68277b37fad8ae1fdf0b5530f
LogPath            : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_3_9_dbbf13b3-efd0-40dc-bf39-5424ae7ebaf4_Tmp-G_Out-J.txt
StatLogPath        : C:\Users\User\.chia\mainnet\plotter\PlotoSpawnerLog_21_5_3_9_dbbf13b3-efd0-40dc-bf39-5424ae7ebaf4_Tmp-G_Out-J@Stat.txt
CompletionTime     : None
CompletionTimeP1   : 15959.216 seconds. CPU (116.290%) Fri May 21 07:35:26 2021
CompletionTimeP2   : 8258.516 seconds. CPU (75.070%) Fri May 21 09:53:05 2021
CompletionTimeP3   :
CompletionTimeP4   :
EndDate            :
```

**Comment by TobiasJu** [comment]
So i removed all jobs that i could with:

` Stop-PlotoJob -JobId 345de945-fff5-43a3-8e0d-ae185ad731d7`

``
But the last one i could not delte, also it's there 4 times.
```
PS C: \ Users \ User \ Downloads \ Ploto-1.0.9.3> Start-PlotoSpawns
PlotoManager @ 05/21/2021 7:36:00 PM: Ploto Manager started.
PlotoManager @ 21.05.2021 19:36:00: Loading config from C: \ Users \ me \ .chia \ mainnet \ config ...
PlotoManager @ 05/21/2021 7:36:00 PM: InputAmountToSpawn: 200
PlotoManager @ 21.05.2021 19:36:00: Interval to wait: 5
PlotoManager @ 21.05.2021 19:36:00: Detected aborted jobs. Removing them ...
PlotoRemoveAbortedJobs @ 5/21/2021 7:36:00 PM: Found aborted Jobs to be deleted: 345de945-fff5-43a3-8e0d-ae185ad731d7 345de945-fff5-43a3-8e0d-ae185ad731d7 345de5d9-fff8e -8e0d-ae185ad731d7
PlotoRemoveAbortedJobs @ 21.05.2021 19:36:00: Cleaning up ...
PlotoStopJob @ 21.05.2021 19:36:00: ERROR: The "Id" parameter cannot be bound. The value "None" cannot be converted to the type "System.Int32". Error: "The input string is in the wrong format."
Get-ChildItem: The argument cannot be processed because the value of the "path" argument is NULL. Change the value of the "path" argument to a value other than NULL.
In C: \ Users \ User \ Downloads \ Ploto-1.0.9.3 \ Ploto.psm1: 1191 characters: 25
+ $ FileArrToDel = Get-ChildItem $ job.TempDrive | Where-Object {...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo: InvalidArgument: (:) [Get-ChildItem], PSArgumentNullException
    + FullyQualifiedErrorId: ArgumentNull, Microsoft.PowerShell.Commands.GetChildItemCommand

PS C: \ Users \ User \ Downloads \ Ploto-1.0.9.3>
```

**Comment by chefsas** [comment]
I'm having the same problem. I had to increase my max parallel jobs to compensate for the ones that are stuck. Every few minutes Ploto thinks there are aborted jobs to cleanup and I see this on my screen (attached).
![Aborted](https://user-images.githubusercontent.com/74569709/119217151-cbfe3b00-ba9d-11eb-8fe0-cc4331f50878.png)

**Comment by tydeno** [comment]
Okay, thanks for reporting. What output do you get when you run "Get-PlotJobs"? Are there any aborted jobs?

**Comment by tydeno** [comment]
Okay I‘ve spotted the issue. TempDrive is not correctly in the Output for reasons. Thats why Stop-PlotoJob cant find the tempDrive and delete the files. I‘ll check it.

**Comment by tydeno** [comment]
As a workaround, you may delete the aborted jobs manually (.tmp and .logs)

**Comment by tydeno** [comment]
I also see t2 drive is set to 2 due to a bug. I got that bug aswell, but was able to fix it in 1.9.4 if I remember correctly.
Could you try updating to Ploto 1.9.4?
